docs/troubleshooting/troubleshooting.md

```diff
@@ -1,4 +1,69 @@
+!!! note "In progress"
+    As we continue expanding our troubleshooting guide, we understand that issues may arise during your usage of Claudie. Although the guide is not yet complete, we encourage you to create a [GitHub issue](https://github.com/berops/claudie/issues) if you encounter any problems. Your feedback and reports are highly valuable to us in improving our platform and addressing any issues you may face.
+
+## Claudie cluster not starting
+Claudie relies on all services to be interconnected. If any of these services fail to create due to node unavailability or resource constraints, Claudie will be unable to provision your cluster.
+
+1. Check if all Claudie services are running:
+
+    ```bash
+    kubectl get pods -n claudie
+    ```
+
+    ```text
+    NAME                           READY   STATUS      RESTARTS        AGE
+    ansibler-5c6c776b75-82c2q      1/1     Running     0               8m10s
+    builder-59f9d44596-n2qzm       1/1     Running     0               8m10s
+    context-box-5d76c89b4d-tb6h4   1/1     Running     1 (6m37s ago)   8m10s
+    create-table-job-jvs9n         0/1     Completed   1               8m10s
+    dynamodb-68777f9787-8wjhs      1/1     Running     0               8m10s
+    frontend-5755b7bc69-5l84h      1/1     Running     0               8m10s
+    kube-eleven-64468cd5bd-qp4d4   1/1     Running     0               8m10s
+    kuber-698c4564c-dhsvg          1/1     Running     0               8m10s
+    make-bucket-job-fb5sp          0/1     Completed   0               8m10s
+    minio-0                        1/1     Running     0               8m10s
+    minio-1                        1/1     Running     0               8m10s
+    minio-2                        1/1     Running     0               8m10s
+    minio-3                        1/1     Running     0               8m10s
+    mongodb-67bf769957-9ct5z       1/1     Running     0               8m10s
+    scheduler-654cbd4b97-qwtbf     1/1     Running     0               8m10s
+    terraformer-fd664b7ff-dd2h7    1/1     Running     0               8m9s
+    ```
+
+1. After verifying if any pods in a failed state, you can check the logs of the frontend service for each service. Typically, if all Claudie pods are scheduled but cluster creation fails, it may be due to missing permissions on the credentials used for Claudie providers.
+
+    ```bash
+    kubectl -n claudie logs -l app.kubernetes.io/name=frontend
+    ```
+
+    ```text
+    6:04AM INF Using log with the level "info" module=frontend
+    6:04AM INF Frontend is ready to process input manifests module=frontend
+    6:04AM INF Frontend is ready to watch input manifest statuses module=frontend
+    ...
```

Comment:
Here you are merging logs from Frontend and from Terraformer. This may be misleading. Maybe show an error in the Frontend and then explain that based on the error message, the more concrete error will be in Terraformer.